### DISCUSSION TITLE
Release 0.12 planning

### DISCUSSION THREAD

**Discussion by ncdc** [kcp-dev/kcp#2891] [General]
What would you all like to see make it into the next release, 0.12

https://github.com/kcp-dev/kcp/milestone/14

At present, the only item included is reverse permission claims.

**Comment by stevekuznetsov** [comment]
@pdettori @MikeSpreitzer would love to get your inputs here!

**Comment by pdettori** [comment]
@stevekuznetsov having a way to configure upsyncable resources as per https://github.com/kcp-dev/kcp/issues/2804 would be really helpful for some use cases I have been looking into.

**Reply by davidfestal** [reply]
Not a priority anymore, do you confirm @pdettori ?

**Reply by pdettori** [reply]
@davidfestal, yes, agreed.

**Comment by nrb** [comment]
I know the work's already in flight, but to be explicit I propose that including Kube 1.26 be a blocker so we don't get too far behind on upstream releases

**Comment by mjudeikis** [comment]
Started working on consolidating modules (api, client into SDK) https://github.com/kcp-dev/kcp/pull/2894 . Thinking if it is worth trying to get this in as the client was merged, but we already agreed this is not quite the right way so if we get this in we avoid publishing the client module altogether and jump directly to SDK one?

But for this to merge we will need 2 PRs: Merge first PR, and move `apigen` to new SDK module as it has a dependency on API too.

**Comment by lionelvillard** [comment]
@davidfestal do you think we can get https://github.com/kcp-dev/kcp/pull/2910 in?
